spec/imsc-hrm.html

```diff
@@ -433,8 +433,8 @@ <h2>Architecture</h2>
       <a>Intermediate Synchronic Documents</a> E<sub>i</sub>:</p>
       <ul>
         <li><a>non-empty ISDs</a> are processed using a simple double buffering model: while a <a>non-empty ISD</a>
-        E<sub>n</sub> is being painted into a Back Buffer, the previous <a>non-empty ISD</a> E<sub>m</sub> is available
-        for display in a Front Buffer; and</li>
+        E<sub>n</sub> is being painted into a Back Buffer by the <dfn>Presentation Compositor</dfn>, the previous <a>non-empty
+        ISD</a> E<sub>m</sub> is available for display in a Front Buffer; and</li>
```

Comment:
Although the term Presentation Compositor is now wrapped in `<dfn>`, it doesn't really help the reader understand what the Presentation Compositor _is_. I was hoping for a definition that would define it at least a little, for example by saying it is a conceptual or hypothetical Presentation Processor assumed to be capable of rendering ISDs from conformant IMSC Document Instances and rasterising the output into an array of pixels.

Reply:
The presentation compositor is defined in the context of the operations it performs and I do not think it is necessary (or useful) to create a separate definition for it -- which would also mean creating a definition for all the processing blocks of the HRM.

Reply:
That's a good point - maybe a standalone sentence along the lines:

>Each of the terms Presentation Compositor, Glyph Renderer, Glyph Copier, Image Decoder, and Image Copier is defined solely by the algorithmic requirements defined for it in this specification.

would clarify?

Reply:
Ok with me. As a note or in the figure caption or ...?

Reply:
See https://github.com/w3c/imsc-hrm/pull/61/commits/b03fcfc8557b6f766ad792944d35fb2609079cac